include/dxc/Support/WinAdapter.h

```diff
@@ -44,6 +44,12 @@
 #define CoTaskMemAlloc malloc
 #define CoTaskMemFree free
 
+// Windows-specific heap functions. Use malloc/realloc/free instead.
+#define HeapAlloc(hHeap,dwFlags, nBytes) malloc(nBytes)
+#define HeapReAlloc(hHeap, dwFlags, voidPtr, nBytes) realloc(voidPtr, nBytes)
+#define HeapFree(hHeap, dwFlags, voidPtr) free(voidPtr)
+#define HeapCreate(flags, nBytes, maxSize) malloc(nBytes)
+
```

Comment:
In https://github.com/google/DirectXShaderCompiler/commit/4002531aaf84027afbe8714733288678e1a95480 , you converted direct mallocs, frees and reallocs to Allocator:: versions. Do you want to do the same here?

Reply:
That was because the class was templated with an Allocator:
```cpp
 template <class T, class Allocator = CAllocator> class CHeapPtrBase
```

In this case, we are dealing with functions inside
```cpp
struct NoSerializeHeapMalloc : public IMalloc
```
which don't have that.

Reply:
Ah. Clearly I wasn't paying enough attention to the previous request and assumed it was some global static class.